qrcode.go

```diff
@@ -0,0 +1,45 @@
+package xendit
+
+import "time"
+
+// QRCodeTypeEnum constants are the available QR Code type
+type QRCodeTypeEnum string
+
+// This consists the values that QRCodeTypeEnum can have
+const (
+	DynamicQRCode QRCodeTypeEnum = "DYNAMIC"
+	StaticQRCode  QRCodeTypeEnum = "STATIC"
+)
```

Comment:
Can we follow Go convention for naming enum?

Reply:
Probably just `QRCodeType` would do

Reply:
Done sir